docs/install/kubernetes.md

```diff
@@ -317,4 +318,66 @@ helm install -n mlrun --values ~/tmp/mlrun-ce-values.yaml mlrun-ce mlrun-ce/mlru
 
 ```{admonition} Note
 If your values have fixed mlrun service versions (e.g.: mlrun:1.3.0) then you might want to remove it from the values file to allow newer chart defaults to kick in
-```
\ No newline at end of file
+```
+
+## Storing artifacts in AWS S3 storage
+
+MLRun CE is using a Minio service as shared storage for artifacts, and accesses it using S3 protocol. This means that
```

Comment:
MLRun CE uses a Minio service as shared storage for artifacts, and accesses it using S3 protocol. This means that
any path that begins with `s3://` is automatically directed by MLRun to the Minio service. The default artifact
path is also configured as `s3://mlrun/projects/{{run.project}}/artifacts` which is a path on the `mlrun` bucket in the
Minio service.